sdk/core/core/src/az_span.c

```diff
@@ -129,7 +129,9 @@ AZ_NODISCARD az_result az_span_to_uint32(az_span span, uint32_t* out_number)
   return AZ_OK;
 }
 
-AZ_NODISCARD az_result az_span_find(az_span source, az_span target, az_span* out_span)
+#define AZ_SPAN_FIND_TARGET_NOT_FOUND -1
```

Comment:
Does this goes into the .h file?

Reply:
No, it will be removed. The public API will return -1, and that's what should be checked by developers.